### DISCUSSION TITLE
API timeout issue in thunderClient

### DISCUSSION THREAD

**Discussion by mubeentajss** [thunderclient/thunder-client-support#1347] [General]
Hello team,

One of my API call request is taking infinite time to respond in thunderClient output, same thing I checked in postman its responding in a minute I know this specific API call is taking time to respond but after sometime postman is responding with output message where as in thunderClient it is keep spinning infinite time and throwing any response output, am not sure is this is something related to settings kindly help us on this issue thanks !

**Comment by rangav** [comment]
Share the video of request of api you have issue 
or
 use any other api client that works for you

**Comment by rangav** [comment]
What is your team size and your company name?

**Comment by mubeentajss** [comment]
thunderClient works for our use cases and we were about to purchase thunderclient liscense thats the reason we wanted to make sure all the API works fine or not.

**Reply by rangav** [reply]
Then sign up for trial account 

also make sure you have read free version terms
https://www.thunderclient.com/terms#free-plan
